### ISSUE TITLE
Install guide

### ISSUE THREAD

**Issue by i3bitcoin** [neo-project/neo-cli#6]
Hi,
Is there any installation guide for Ubuntu server?

Thanks.

**Comment by erikzhang** [comment]
Is this doc helpful to you?
http://docs.neo.org/en-us/node/setup.html

**Comment by redi78** [comment]
Hi, i want run the node on Debian 8, but after install i get the message: Segmentation Fault
Do you have any idea? Where can i find the file libleveldb.dll?

Thanks

**Comment by i3bitcoin** [comment]
Thanks) I've missed your reply, found the manual for Antshares here https://github.com/neo-project/neo/wiki/Bookkeeping-Node-Deployment

Anyway, I got an error
A fatal error was encountered. The library 'libhostpolicy.so' required to execute the application was not found in '/root/neo'.

Ubuntu 16.04.3 LTS

**Comment by i3bitcoin** [comment]
Finally I've tried to build neo-cli from the source code and got the same error as **redi78**
Segmentation fault

**Comment by erikzhang** [comment]
You need to install leveldb.
```
yum install leveldb-devel
```

**Comment by erikzhang** [comment]
Maybe you will need some other dependencies:
![image](https://user-images.githubusercontent.com/10204923/29246933-81a4ef24-8039-11e7-8ba2-5de1308fdbd1.png)

**Comment by erikzhang** [comment]
See the step-by-step CentOS instalation guide in #13

**Comment by redi78** [comment]
Hi, i installed all the dependencies but i get the Segmentation fault error again. Any other ideas?
BR Rene

**Comment by jonatanblockchain** [comment]
I've just updated installations guide version 2.1.0 on a fresh installation and it's working well.
Step-by-step Centos and Ubuntu installation guide #13

**Comment by vvhof** [comment]
I think this is what @redi78 is referring to.

``` 
$ dotnet -d neo-cli.dll
Telemetry is: Enabled
No executable found matching command "dotnet-neo-cli.dll"
```

**Comment by scottemick** [comment]
I've compiled leveldb and I'm trying to compile neo-cli on windows.  I'm getting.  
System.DllNotFoundException: 'Unable to load DLL 'libleveldb': The specified module could not be found. (Exception from HRESULT: 0x8007007E)' at line 714 in MainService.cs

I'm not sure what I should do.  In the json config DataDirectoryPath is just set to Chain.  I tried creating a Chain folder and copying the libleveldb.lib in there, that didn't work
